membership-common/src/test/scala/com/gu/memsub/subsv2/services/TestCatalog.scala

```diff
@@ -27,7 +27,7 @@ object TestCatalog {
   val contributorPrpId = ProductRatePlanId("asdfasdf")
   val supporterPlusPrpId = ProductRatePlanId("8ad08cbd8586721c01858804e3275376")
   val tierThreePrpId = ProductRatePlanId("8ad097b48ff26452019001cebac92376")
-  val guardianLightPrpId = ProductRatePlanId("8a12831492c341730192dd1c39207038")
+  val guardianAdLitePrpId = ProductRatePlanId("8a12831492c341730192dd1c39207038")
```

Comment:
I think this ID needs to change as well doesn't it?

Reply:
[Done](https://github.com/guardian/members-data-api/pull/1111/commits/18a04b865d2e29bf91991ac2913bf8e39533ad69)